app/style/components/group-video-grid.less

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+.video-grid {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+
+  &__element {
+    position: absolute;
+    width: 50%;
+    height: 50%;
+    border: solid 2px #fff;
+  }
+  &__element0 {
+    top: 0;
+    left: 0;
+  }
+  &__element1 {
+    bottom: 0;
+    left: 0;
+  }
+  &__element2 {
+    top: 0;
+    right: 0;
+  }
+  &__element3 {
+    right: 0;
+    bottom: 0;
+  }
+  &__element--full-height {
+    height: 100%;
+  }
+  &__element--full-size {
+    width: 100%;
+    height: 100%;
+  }
+  &__element--empty {
+    overflow: hidden;
+    width: 0;
+    height: 0;
+  }
```

Comment:
In theory we could also do some nesting with the `&__element*` as well.. but I'm not sure if @AndyLnd will agree :)

Reply:
I agree, that's why we use a css preprocessor :)
also i do it the same way.

Reply:
Cool! Letz do diz then!